crates/cibse_guide_e/src/chapter_6/equation_6_7.rs

```diff
@@ -0,0 +1,20 @@
+pub mod openfire_runner;
+
+pub fn heat_release_rate_flashover(a_vo: f64, h_o: f64) -> f64 {
+    return 600.0 * a_vo * h_o.powf(0.5);
+}
+
+pub fn equation(q_f: String, a_vo: String, h_o: String) -> String {
+    format!("{} = 600 \\space {} \\sqrt {{{}}}", q_f, a_vo, h_o,)
```

Comment:
```suggestion
    format!("{} = 600 \\cdot {} \\sqrt {{{}}}", q_f, a_vo, h_o,)
```
This looks bad when it is numbers and not symbols